shared_python/Tags.py

```diff
@@ -6,6 +6,7 @@
 from pymysql import cursors, OperationalError
 
 from shared_python.Sql import Sql
+from shared_python.ValidateTags import ValidateTags
```

Comment:
I'll be a bit picky here and argue that [classes should have names that are nouns](https://thecoderoad.blog/2020/03/29/clean-code-naming-conventions/). The class itself doesn't validate the tags; an instance of the class is a <i>thing</i> that provides methods that validate tags. A logical alternative name would therefore presumably be `TagValidator` (though open to discussion if you have something better!)

Reply:
That is a valid point, so now ValidateTags is now named TagValidator!